### PR TITLE
Gat/ssqc report fixes

### PR DESCRIPTION
**Pull request by GatKrieger**
After Omry's comments

### CODE REVIEW

File: ugvc/scripts/sorter_to_h5.py [hunk 1]
```diff
@@ -119,6 +136,18 @@ def sorter_to_h5(
     ]
     h5_dict["RawWgsMetrics"] = {k: h5_dict["RawWgsMetrics"][k] for k in key_order}
 
+    # Q20_PF_BASES and Q30_PF_BASES
+    h5_dict["QualityYieldMetricsFlow"]["Q20_BASES"] = round(
```
Comment (doron-st) [inline ugvc/scripts/sorter_to_h5.py:140]: I think it's concerning to rename he percentage of pass-filter bases as just bases.
Do we have another choice?

**Review by doron-st — COMMENTED** [review]
just one comment

File: ugvc/scripts/sorter_to_h5.py (hunk 1 above)
Comment (GatKrieger) [reply ugvc/scripts/sorter_to_h5.py:140]: That's what we should do according to Omer.

**Review by doron-st — APPROVED** [review]
LGTM

**Pull request merged**